cmd/corald/main.go

```diff
@@ -48,7 +48,7 @@ func main() {
 		writeTimeout = 30 * time.Second
 	)
 
-	if err := app.Run(":4001", routes.API(), readTimeout, writeTimeout); err != nil {
+	if err := app.Run(":3000", routes.API(), readTimeout, writeTimeout); err != nil {
```

Comment:
I'm going to look at that now as it seems that the apps are not getting the port from the configuration.